feature/webview/src/main/java/com/nexters/misik/webview/WebViewScreen.kt

```diff
@@ -66,6 +93,41 @@ fun WebViewScreen(
         )
     }
 
+    LaunchedEffect(initializedUrl) {
+        if (initializedUrl.isNotEmpty()) {
+            webView.loadUrl(initializedUrl)
+            Timber.d("WebViewScreen_LoadingUrl: $initializedUrl")
+        }
+    }
+
+    DisposableEffect(view) {
+        val listener = View.OnApplyWindowInsetsListener { v, insets ->
+            val imeBottom = if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.R) {
+                insets.getInsets(WindowInsetsCompat.Type.ime()).bottom
+            } else {
+                WindowInsetsCompat.toWindowInsetsCompat(insets).systemWindowInsetBottom
+            }
+
+            viewModel.updateKeyboardHeight(imeBottom)
+
+            insets
+        }
+
+        view.setOnApplyWindowInsetsListener(listener)
+
+        onDispose { view.setOnApplyWindowInsetsListener(null) }
+    }
+
+    // 키보드 높이 변화 시 웹에 전달
+    LaunchedEffect(keyboardHeight) {
+        if (keyboardHeight > 0) { // 키보드가 올라왔을 때만 전달
+            val jsCode =
+                makeKeyboardHeightResponse("receiveKeyboardHeight", keyboardHeight.toString())
+            webView.evaluateJavascript(jsCode, null)
+            Timber.d("WebViewScreen_sendKeyboardHeight", jsCode)
+        }
+    }
```

Comment:
여러 LaunchedEffect 및 DisposableEffect 블록이 중첩되어 가독성이 떨어지는 것 같습니다. 각 효과(키보드 인셋 업데이트, JS 코드 평가 등등)를 별도의 헬퍼 Composable 함수로 분리하고, 각 효과의 역할을 명확하게 주석으로 달면 코드의 유지보수성 크게 향상될 것 같아요!
```suggestion
    @Composable
fun KeyboardInsetsListener(onKeyboardHeightChanged: (Int) -> Unit) {
    val view = LocalView.current
    DisposableEffect(view) {
        ...
    }
}

// 키보드 높이 업데이트를 별도 함수로 분리
// 키보드 높이가 변경되면 자바스크립트로 전달

...
```